onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -94,13 +85,18 @@ class NodeRepo {
     return node_repo;
   }
 
-  onnxruntime::Status AddNode(const onnxruntime::OpKernel* kernel, NodePtr node_ptr) {
+  onnxruntime::Status AddNode(const onnxruntime::OpKernel* kernel, NodePtr node_ptr, ArgPtrs& args) {
```

Comment:
ArgPtrs&&

Reply:
perhaps even `onnxruntime::Status AddNode(const onnxruntime::OpKernel* kernel, NodePtr&& node_ptr, ArgPtrs&& args)` to be consistent

Reply:
Done.